### PR TITLE
BMXSDOC-4979 Explain how to use Rule Tasks, User Tasks, and Script tasks and include other constructors

### PR DESCRIPTION
**Pull request by emmurphy1**
This jira includes changes for Designing business process in Business central. Details in epic:
[BXMSDOC-4937](https://issues.redhat.com/browse/BXMSDOC-4937)

### CODE REVIEW

File: doc-content/enterprise-only/project-data/action-scripts-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[id='action-scripts-proc_{context}']
+
+= Defining an action script
+
+Action scripts are code snippets that are associated with a script task. They specify the action of the script task. You can include global and process variables in your action scripts.
+
+You can write action scripts in Java, JavaScript, and MVEL. Note that MVEL accepts any valid Java code and additionally provides support for nested access of parameters. For example, the MVEL equivalent of the Java call `person.getName()` is `person.name. MVEL also provides other improvements over Java and MVEL expressions are generally more convenient for business users.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/project-data/action-scripts-proc.adoc]: is `person.name.  is missing a closing tag.

File: doc-content/enterprise-only/project-data/variables-con.adoc [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+== Defining process variables
+Process variables are defined as properties in the BPMN2 definition file and are visible within the process instance. They are initialized at process creation and destroyed on process completion.
+
+A process variable is a variable that exists in a process context and can be accessed by its process or its child elements. Process variables belong to a particular process instance and cannot be accessed by other process instances.
+Every process variable defines its ID and item subject reference: the ID serves as the variable name and must be unique within the process definition.
+The item subject reference defines the data type the variable stores.
+
+Process variables are initialized when the process instance is created.
+Their value can be changed by the process activities using the Assignment, when the global variable is associated with the local Activity context, local Activity variable, or by a direct call to the variable from a child context.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/project-data/variables-con.adoc]: Minor, but shouldn't the second sentence be right after the 1st?

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-script-task-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
+[id='create-script-task-proc']
+
+= Creating script tasks
+Script tasks are used to execute a piece of code written in Java, JavaScript, or MVEL. They contain code snippets that specify the action of the script task. You can include global and process variables in your  scripts.
+
+You can write action scripts in Java, JavaScript, and MVEL. Note that MVEL accepts any valid Java code and additionally provides support for nested access of parameters. For example, the MVEL equivalent of the Java call `person.getName()` is `person.name. MVEL also provides other improvements over Java and MVEL expressions are generally more convenient for business users.
```
Comment (michelehaglund) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-script-task-proc.adoc]: Same comment as above. Missing end tag for is `person.name.

**Review by michelehaglund — APPROVED** [review]
Minor comments added.

File: doc-content/enterprise-only/project-data/variables-con.adoc (hunk 2 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/project-data/variables-con.adoc]: Yes, fixed.

**Pull request merged**